README.md

```diff
@@ -1 +1,23 @@
-Coming soon
+# PostgreSQL Kubernetes Operator
+
+## Overview
+
+This repository hosts the PostgreSQL Kubernetes Operator.
+
```

Comment:
```suggestion
```

The overview section doesn't seem to add much value here :)

Reply:
I agree. I removed it as we already have a better overview in the description section.